src/app/loadout-builder/filter/ModPicker.tsx

```diff
@@ -276,21 +279,21 @@ function ModPicker({
     category === ModPickerCategories.general || category === ModPickerCategories.seasonal;
 
   const footer = Object.values(lockedArmor2ModsInternal).some((f) => Boolean(f?.length))
-    ? ({ onClose }) => (
+    ? () => (
         <ModPickerFooter
           defs={defs}
           categoryOrder={order}
           lockedArmor2Mods={lockedArmor2ModsInternal}
           isPhonePortrait={isPhonePortrait}
-          onSubmit={(e) => onSubmit(e, onClose)}
```

Comment:
This `onClose` stuff was done for a very specific reason - it causes the Sheet to animate closed, and *then* fire an `onClose` event which can be used to actually hide it. If you just hide it directly using `modPicker.isOpen`, it'll instantly disappear with no animation. Can we put all this back?

Reply:
Yeah can do, I was just trying to avoid passing the handler through all the components but it's probably necessary.